corehq/form_processor/utils.py

```diff
@@ -33,3 +35,11 @@ def _inner(*args, **kwargs):
         return _wrap(obj)
 
     return _inner
+
+
+def get_backend():
```

Comment:
As mentioned further up I think it will have to be domain based (probably using a toggle). Probably a similar pattern to what @czue did here: https://github.com/dimagi/commcare-hq/blob/multi-test-cleanup/corehq/ex-submodules/casexml/apps/phone/restore.py#L382-382

```python
XFormInterface(domain).get_xform(xform_id)
```